tests/vcd_user.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import string
+import random
+
+from pyvcloud.vcd.org import Org
+from pyvcloud.vcd.test import TestCase
+
+
+class TestUser(TestCase):
+    def test_create_user(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        role = org.get_role(self.config['vcd']['role_name'])
+        role_href = role[0]['href']
+        user_name = self.config['vcd']['user_name'].join(random.sample(string.ascii_lowercase, 8))
+        user = org.create_user(user_name, "password", role_href, "Full Name", "Description", "xyz@mail.com",
+                               "408-487-9087", "test_user_im", "xyz@mail.com", "Alert Vcd:",is_enabled=True)
+        assert user_name == user.get('name')
+
```

Comment:
Shouldn't we be doing a cleanup after the test and delete the user that we just created for test purposes?
In case delete user is yet to be implemented. Please have a TODO item in here.

Reply:
Added TODO.